about.html

```diff
@@ -0,0 +1,170 @@
+          <span class="sub-heading">"Hello! Sharing World"</span>
+          <h1 class="primary-heading">Creative Commons Global Summit 2015</h1>
+          <p class="landing-text">
+            The Global Summit brings together the community of experts,
+            academics, and activists who comprise the Creative Commons affiliate
+            network in a different country every two years. This year, we hope
+            to expand our invitation list including organizations and
+            individuals who want to work with us on shared projects that advance
+            the cause of the Commons, free culture and open knowledge.
+          </p>
+          <p class="content">
+            Please contact via our email if you have any further questions about
+            CC Global Summit 2015!
+          </p>
+          <a class="email">thisorganization@email.com</a>
+        </div>
+      </div>
+    </section>
+
+    <section class="logo-section">
+      <div class="container">
+        <h2 class="secondary-heading">
+          Creative Common <br />
+          Global Summit 2015 Logo
+        </h2>
+        <span class="underline"></span>
+        <p class="content">
+          The logo of CC Global summit 2015 was decided through the logo
+          competition from 8.July
+        </p>
+        <div class="logo-box">
+          <img src="./images/cc-global.png" alt="logo image" />
+        </div>
+      </div>
+    </section>
+
+    <section class="events-section">
+      <div class="container">
+        <h2 class="secondary-heading">See the past CC Global summits</h2>
+        <span class="underline"></span>
+        <p class="content">
+          Take a look at the last two CC Global summits which took place in
+          Buenos Aires and Warsaw.
+        </p>
+
+        <div class="events">
+          <div class="event">
+            <p class="date">2013</p>
+            <p class="name">CC Global Summit 2013</p>
+          </div>
+
+          <div class="event e-2">
+            <p class="date">2011</p>
+            <p class="name">CC Global Summit 2011</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="partners-section">
+      <div class="container">
+        <h5 class="secondary-heading">Partner</h5>
+        <p class="underline"></p>
+        <div class="partners">
+          <img src="./images/partners/mozilla.png" alt="" />
+          <img src="./images/partners/airbnb.png" alt="" />
+          <img src="./images/partners/ibm.png " alt="" />
+          <img src="./images/partners/amazon.png" alt="" />
+          <img src="./images/partners/netflix.png" alt="" />
+        </div>
+      </div>
+    </section>
+
+    <footer>
+      <div class="container">
+        <div class="footer-logo">
+          <img src="./images/logo.png" alt="" />
+        </div>
+        <div class="footer-content">
+          Creative Commons Korea. Some Rights Reserved.
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] Kindly consider the footer in about page has a dark background as exits in the design guidelines.